Cargo.lock

```diff
@@ -198,6 +198,22 @@ version = "2.3.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "6776fc96284a0bb647b615056fc496d1fe1644a7ab01829818a6d91cae888b84"
 
+[[package]]
+name = "block-buffer"
```

Comment:
So many new dependencies. Are they all introduced because of the bigint library migration? Would it make sense for us to stay on numbigint for now?

Reply:
I'm not sure, but we _are_ staying on num-bigint, aren't we? I'm not using the malachite feature.

Reply:
It does look like `block-buffer` comes from `malachite`. But does this actually affect the binary? I can also just remove the `malachite` stuff in a separate PR.

Reply:
Oh, I think `rustpython_format` doesn't have this properly gated as a feature. I'll fix this separately.

Reply:
Ok, fixed this. No new deps.

Reply:
Thank you!